Cargo.toml

```
@@ -21,3 +24,7 @@ criterion = { version = "0.4", features = ["html_reports"] }
 [[bench]]
 name = "full_benchmark"
 harness = false
+
+[[example]]
+name = "live_plotting"
+required-features = ["live-plotting"]
```

Add an empty line at the end of the file
```suggestion
required-features = ["live-plotting"]

```